### PR TITLE
test(pl-param-resource): Add test for Pipeline parameters & resources

### PR DESCRIPTION
**Pull request by abhinandan13jan**
Test Pipeline Parameters & Resources Tabs
https://issues.redhat.com/browse/ODC-2579
https://issues.redhat.com/browse/ODC-2580

### CODE REVIEW

File: frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  });
+
+  it('Should render the correct parameters', () => {
+    const formWrapper = plFormWrapper.find('Formik').shallow();
+    const params = formWrapper.find('PipelineParameters').prop('values');
+    expect(params).toEqual(expected);
+  });
+});
+
+describe('Pipeline details parameter tab for pipeline with parameters in place', () => {
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx]: Why two describe blocks? It looks like you're testing the same component but split up the tests for no gain and duplicated some effort.

File: frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PipelineForm from '../../pipeline-form/PipelineForm';
+import PipelineParameters from '../PipelineParameters';
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx]: This file is actually a "`PipelineForm` test"... we should just validate it shows a `Formik` component that somewhere beneath it shows `PipelineParameters` component; hopefully that doesn't couple to the Formik implementation and cause erroneous test breaks if we update `Formik`... I think it's okay, but we have to be careful not to assume anything about the `Formik` component structure and just know it needs to preset our rendered component at some point.

Comment (andrewballantyne) [reply frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineParameters.spec.tsx]: You could probably also add a test to validate the `initialValues` prop in `Formik` but it shouldn't go any further than that.

File: frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineResources.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import PipelineForm from '../../pipeline-form/PipelineForm';
+import PipelineResources from '../PipelineResources';
```
Comment (andrewballantyne) [inline frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/__tests__/PipelineResources.spec.tsx]: Same issue here... we are repeating ground with tests and that is the last thing you want to do as it'll make multiple tests flag failed when it's unrelated to them.

**Comment by abhinandan13jan** [comment]
Closing this PR out as its stale now. Will retake after the changes associated to the section being tested  are in.

**Pull request closed without merging**